site2/docs/io-file-source.md

```diff
@@ -105,7 +105,13 @@ Here is an example of using the File source connecter.
     $ curl -O https://mirrors.tuna.tsinghua.edu.cn/apache/pulsar/pulsar-{version}/connectors/pulsar-io-file-{version}.nar
     ```
 
-6. Start the File source connector.
+6. Copy the File source connector to the container. 
+
+    ```bash
+    $ docker cp pulsar-io-file-{version}.nar pulsar-standalone:/pulsar/
```

Comment:
Copy it to the `connectors` folder, then restart the container

```
docker cp pulsar-io-file-{version}.nar pulsar-standalone:/pulsar/connectors/
docker restart pulsar-standalone
```

Reply:
Hi @Haaroon any feedback?

Reply:
Hi @tuteng, @Haaroon has incorporated your comment, any other technical comments?